source/getting-started-edge/index.rst

```diff
@@ -89,6 +89,142 @@ resource usage. The dashboard looks  something like this:
 
    An overview of your project's current resource usage
 
+Setting up CLI
+--------------
+
+Right now, edge device reservations can only be made through the command line 
+interface. This means that you must follow the instructions
+`here <../technical/cli.html>`_ to set up your blazar client. If you have
```

Comment:
Using refs is more sustainable, as you can move pages around. I think it works like this:

```rst
`here <ref-id>`_
```

there are examples elsewhere I am pretty sure.